docs/developer-docs/multi-chain/bitcoin/using-btc/runes.mdx

```diff
@@ -0,0 +1,11 @@
+---
+keywords: [advanced, bitcoin, tutorial, btc, runes, schnorr, taproot]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Runes
+
+<MarkdownChipRow labels={["Advanced", "Bitcoin", "Tutorial"]} />
+
+## Overview
```

Comment:
@jessiemongeon1 Did you intend to add more to this page?

Reply:
Yes, I haven't gotten a chance to write this page yet.